config/defaults.go

```diff
@@ -9,7 +9,8 @@ import (
 const (
 	// DefaultListenAddress is a default listen address for P2P client.
 	DefaultListenAddress = "/ip4/0.0.0.0/tcp/7676"
-	Version              = "0.4.0"
+	// Version is the current rollkit version and is used for checking RPC compatibility??
```

Comment:
This should be bumped to v0.12.0 right with the mainnet release?  Also is it accurate that this is for RPC compatibility? Or is there another use for the version?

Reply:
Is this the celestia-node version? With DA extraction, my understanding is that we’re avoiding using any DA layer specific versions, correct?

Reply:
created #1323